src/Symfony/Bridge/PhpUnit/ClockMock.php

```diff
@@ -100,12 +100,12 @@ public static function hrtime($asNumber = false)
     {
         $ns = (self::$now - (int) self::$now) * 1000000000;
 
+
```

Comment:
That additional line does not look right.

Reply:
Indeed, this is weird :thinking: Reverted this one as well as your second comment